io-engine/src/lvs/lvol_snapshot.rs

```diff
@@ -691,4 +713,44 @@ impl SnapshotOps for Lvol {
             .filter(|b| b.is_snapshot_clone().is_some())
             .collect::<Vec<Lvol>>()
     }
+    /// Check if the snapshot has been discarded.
+    fn is_discarded_snapshot(&self) -> bool {
+        Lvol::get_blob_xattr(self, SnapshotXattrs::DiscardedSnapshot.name())
+            .unwrap_or_default()
+            .parse()
+            .unwrap_or_default()
+    }
+
+    /// During destroying the last linked cloned, if there is any fault
+    /// happened, it is possible that, last clone can be deleted, but linked
+    /// snapshot marked as discarded still present in the system. When
+    /// control plane does the redo of the destroy replica, clone uuid will
+    /// not be found in the system, but discarded snapshot can be searched
+    /// and destroyed.
+    async fn destroy_pending_discarded_snapshot(clone_uuid: &str) {
+        info!("HR: Pending destroy: {:?}", clone_uuid);
+        let bdev = match UntypedBdev::bdev_first() {
+            Some(b) => b,
+            None => {
+                info!("HR: No Device");
```

Comment:
Just wonder whether we need 'info' level here ? Isn't it too verbose?

Reply:
> Just wonder whether we need 'info' level here ? Isn't it too verbose?

It was added for issue issue debugging. Will remove this info here. We have debug and error statement at the end of the function for success and failure case.